docs/reference/services/ext_authz.md

```diff
@@ -0,0 +1,85 @@
+## The `ext_authz` Protocol
+
+By design, the `ext_authz` protocol used by [the `AuthService`](../auth-service) and by [`External` `Filters`](../../filter-reference) is highly flexible. The authentication service is the first external service invoked on an incoming request (e.g., it runs before the rate limit filter). Because the logic of authentication is encapsulated in an external service, you can use this to support a wide variety of use cases. For example:
+
+* Supporting traditional SSO authentication protocols, e.g., OAuth, OpenID Connect, etc.
```

Comment:
Try to have the same verb form for these bullets. So either all ending in -ing or in the infinitive, etc, for consistency

Reply:
Making me reword things that I just cut/pasted from one file to another. :)

Reply:
sigh

Reply:
Fixed, they all use the -ing form now.